src/main/java/org/sonar/scm/git/blame/FileBlamer.java

```diff
@@ -78,21 +78,22 @@ public void initialize(ObjectReader objectReader, StatefulCommit commit) {
 
   /**
    * Blame all remaining regions to the commit
+   *
+   * @param source - commit that will be used to associate blame data with remaining regions in files
    */
-  public void processResult(StatefulCommit source) {
-
+  public void saveBlameDataForFiles(StatefulCommit source) {
     String commitHash = source.getCommit().getName();
     String authorEmail = source.getCommit().getAuthorIdent().getEmailAddress();
     Date commitDate = source.getCommit().getCommitterIdent().getWhen();
     for (FileCandidate sourceFile : source.getAllFiles()) {
       if (sourceFile.getRegionList() != null) {
-        blameResult.process(commitHash, commitDate, authorEmail, sourceFile);
+        blameResult.storeBlameDataForFile(commitHash, commitDate, authorEmail, sourceFile);
       }
     }
   }
 
   public StatefulCommit blameParent(RevCommit parentCommit, StatefulCommit child) throws IOException {
-    List<DiffFile> diffFiles = fileTreeComparator.compute(parentCommit, child.getCommit(), child.getAllPaths());
+    List<DiffFile> diffFiles = fileTreeComparator.findModifiedFiles(parentCommit, child.getCommit(), child.getAllPaths());
```

Comment:
I find the terminology "modifiedFiles" to be a bit confusing . 
Here I think we retrieve files that have "moved" (COPY, RENAME, etc.)

Reply:
In the implementation of this method we are talking about "modifiedFiles".  But yeah it seems like what we are really doing is that we are looking for moved files.

Reply:
and movedFiles is not exact neither, for instance a COPY is not a moved file